js/accordionModel.js

```diff
@@ -0,0 +1,47 @@
+define([
+    'core/js/models/itemsModel'
+], function(ItemsModel) {
+
+    var AccordionModel = ItemsModel.extend({
+
+        defaults: function() {
+            return _.extend({
+                _shouldCollapseItems: true,
+                _toggleSpeed: 200
+            }, ItemsModel.prototype.defaults);
+        },
+
+        initialize: function() {
+            this.resetActiveItems(false);
```

Comment:
this whole initialize function should go back to the ``ItemsModel``, it's standard behaviour for all three components, it'd be nice to have one central place to control it.